packages/hmr/lib/plugins/make-entry-point-plugin.ts

```diff
@@ -50,11 +50,6 @@ export function makeEntryPointPlugin(): PluginOption {
         }
       }
     },
-    closeBundle() {
-      cleanupTargets.forEach(target => {
-        fs.unlinkSync(target);
-      });
-    },
```

Comment:
Why did you remove it because you thought it was unnecessary?

Reply:
Yeah, i was looking if it was used but i haven't seen it anywhere, if you want to leave it unused as before, no problem, tell me i'll back it.

Reply:
This change has nothing to do with TS support, so we'll roll it back and look at that part again, please revert it back. 😄

Reply:
Yeah, indeed i've went too wild with this PR :)

I'll revert it and remove it in other PR, thanks to check me, because i like to push too much changes into 1 PR, that's my bad